crates/biome_js_analyze/src/nextjs.rs

```diff
@@ -0,0 +1,27 @@
+use biome_js_semantic::Binding;
+use biome_js_syntax::JsImport;
+use biome_rowan::AstNode;
+
+/// Represent Next.js libraries.
```

Comment:
Can you explain what "next libraries" means? The implementation shows a `next/script`, which isn't a library. It's a specifier of the `next` package, however this comment seems misleading, because I understood external libraries.

Reply:
sorry you are right, how about call them "Next.js built-in utility"? I would like to point to `next/image`, `next/link` etc.

Reply:
That's definitely better! Good suggestion